python/pyraphtory_jvm/pyraphtory_jvm/ivy.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I believe @fabianmurariu means why do we need this here if we create it fresh.

Reply:
Each build*

Reply:
the ivy file needs to be here, because we use it to upload the sdist to pypi, and it uses the ivy file to build raphtory later, in theory i can delete it as long as always remember to push it when we do a publish

Reply:
i'll add this as a ticket instead of doing it now, so it reminds me to document the build process

Reply:
Make a ticket for it and tag me please